libraries/engage/locations/components/ReserveForm/ReserveFormPhone.jsx

```diff
@@ -0,0 +1,114 @@
+// @flow
+import * as React from 'react';
+import classnames from 'classnames';
+import PhoneInput from 'react-phone-number-input/mobile';
```

Comment:
I figured out a couple of issues which are primarily related to the module.

- One of the acceptance criteria of the story was that we are supposed to show two groups of countries. those which are supported by the shop (we already do this) and the rest of the world. I guess the reason is that you can live in one of the countries where the merchant has stores, but you phone has a foreign SIM card (e.g. when you are on vacation). The module doesn't support this out of the box. We'd need to implement the country select by our own (which is supported by the module) but i'm not sure if we the rest of its functionality.

- When one enters a phone number without international code and submits the form, we persist the number with the internally added area code. The next time one opens the form, a number with area code will be shown. If you now try to clear the number and to add a new one, the input starts to behave weird - it always tries to add an area code. In the console you can see some React errors then.

Reply:
As discussed with PO, this is moved out of scope if this ticket.